_data/jobs.yml

```diff
@@ -1,29 +1,29 @@
-- title: Software Developer - 3rd Party Integration, REST API (m/f/d)
+- title: Software Developer - 3rd Party Integration, REST API (all genders)
   link: https://epages-gmbh.jobs.personio.de/job/202173
   description: Be part of a Scrum team and support us in the conception, development and maintenance of our connectors and the extension of our REST API.
   location: Hamburg/Barcelona
 
-- title: UI Designer HTML/CSS (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/224109
-  description: Drive the visual identity of our products and improve their performance and usability by working closely in an agile team.
-  location: Hamburg/Barcelona
-
-- title: Senior JavaScript Engineer (m/f/d)
+- title: Senior JavaScript Engineer (all genders)
   link: https://epages-gmbh.jobs.personio.de/job/201909
   description: Build a responsive, modular, and highly user interactive web application with us that is easy to scale and to extend.
   location: Hamburg/Barcelona
 
-- title: Ruby Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/202168
-  description: With your knowledge of Ruby and JavaScript, you'll support our Rails team to further develop our crucial in-house applications.
+- title: (Senior) Java Developer (all genders)
+  link: https://epages-gmbh.jobs.personio.de/job/202115
+  description: Join one of our cross-functional Scrum teams and work on a microservice-based architecture for our e-commerce platform.
   location: Hamburg/Barcelona
 
-- title: Technical Product Owner (m/f/d)
+- title: Product Owner (all genders)
   link: https://epages-gmbh.jobs.personio.de/job/226186
   description: Be responsible for the further development of our software and support several Scrum teams with your knowledge.
-  location: Hamburg/Barcelona
+  location: Barcelona
+
+- title: Product Owner (all genders)
+  link: https://epages-gmbh.jobs.personio.de/job/1075968
+  description: Be responsible for the further development of our software and support several Scrum teams with your knowledge.
+  location: Jena
 
-- title: Internship Webdesign (m/f/d) - 6 months or 1 year
-  link: https://epages-gmbh.jobs.personio.de/job/359886
-  description: Grow and learn in our Design & Consult team by advising our international customers and creating individual designs.
-  location: Barcelona
\ No newline at end of file
+- title: Senior User Experience Designer (all genders)
+  link: https://epages-gmbh.jobs.personio.de/job/1137607
+  description: Work closely with product owners, UI designers, UX writers, and developers to develop user-friendly interaction concepts for our products.
```

Comment:
In order to avoid using "developers" and "to develop" maybe we could use another verb like "create", "set up" or "establish"? What do you think?